universal-login-example/test/fixtures/setupSdk.js

```diff
@@ -1,9 +1,26 @@
 import {RelayerUnderTest} from 'universal-login-relayer/build';
 import {createMockProvider} from 'ethereum-waffle';
 import UniversalLoginSDK from 'universal-login-sdk';
+import path from 'path';
+
+const knexConfig = {
+  test: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_test',
+      user:     'postgres',
+      password: ''
+    },
+    migrations: {
+      directory: path.join(__dirname, '../../../universal-login-relayer/migrations'),
+      tableName: 'knex_migrations'
+    }
+  }
+};
```

Comment:
Should be taken from `knexfile.js`, from `test` section (should be developed not test database).
knexfile.js should be individual for `example` project

Reply:
✅